deep_learning/README.md

```diff
@@ -39,3 +39,8 @@
 | Paper| Notes | Author | Summary |
 |:-----:|:-----:|:-----:|:----------:|
 | [Variational Adversarial Active Learning](https://arxiv.org/abs/1904.00370) (ICCV '19) | [HackMD](https://hackmd.io/CxZNGh6dS3m2axmP50iN8g) | [Akshay](https://akshayk07.weebly.com/) | This paper introduces a pool-based active learning strategy which learns a low dimensional latent space from labeled and unlabeled data using a VAE. |
+
+### Feature Detection and Description
+| Paper| Notes | Author | Summary |
+|:-----:|:-----:|:-----:|:----------:|
+| [D2 Net - A Trainable CNN for Joint Description and Detection of Local Features](https://arxiv.org/abs/1905.03561) (CVPR 2019) | [HackMD](https://hackmd.io/@AniketGujarathi/SywvV8iQD) | [Aniket Gujarathi](https://www.linkedin.com/in/aniket-gujarathi/?originalSubdomain=in) | This paper introduces a Deep Learning based approach to solve the problem of local features detection and description using the detect-and-describe approach instead of the traditionally used detect-then-describe approach. |

```

Comment:
For the sake of consistency, change CVPR 2019 to CVPR '19 in the table :sweat_smile: